examples/solver/dumpsad-solver/src/curve.rs

```diff
@@ -0,0 +1,64 @@
+use cosmwasm_std::Uint128;
+
+// curve y = a*1000_000_000 - (b*1000_000_000) / (30 * 1_000_000_000 + x)
+// both SOL and meme has 9 decimals
+// where: y: total meme minted, x: total sol deposited
+pub struct BondingCurve {
+    pub a: Uint128,
+    pub b: Uint128,
+    pub x: Uint128,
+    pub y: Uint128,
+}
+
+impl BondingCurve {
+    pub const PRECISION_MULTIPLIER: Uint128 = Uint128::new(1_000_000_000u128);
+    const DEFAULT_A: Uint128 = Uint128::new(1073000191 * 1_000_000_000u128);
+    const DEFAULT_B: Uint128 = Uint128::new(32190005730 * 1_000_000_000u128);
+
+    pub fn default(x: Uint128, y: Uint128) -> Self {
+        BondingCurve {
+            a: Self::DEFAULT_A,
+            b: Self::DEFAULT_B,
```

Comment:
Can use better names like BASE_LIMIT, QUOTE_LIMIT

Reply:
It's kinda a coefficient but let me think a better name for it